Userland/Applets/Keymap/main.cpp

```diff
@@ -22,7 +22,9 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     window->set_title("Keymap");
```

Comment:
Can't comment on the commit message directly, but the title category here is probably fine with just `LibGUI:`. All the others are just users of that API, and mixing four different categories for that message seems a bit crowded.